docs/ref/class-based-views/generic-editing.txt

```diff
@@ -204,9 +204,9 @@ editing content:
 .. class:: django.views.generic.edit.DeleteView
 
     A view that displays a confirmation page and deletes an existing object.
-    The given object will only be deleted if the request method is ``POST``. If
-    this view is fetched via ``GET``, it will display a confirmation page that
-    should contain a form that POSTs to the same URL.
+    The given object will only be deleted if the request method is ``POST`` or
+    ``DELETE``. If this view is fetched via ``GET``, it will display a
```

Comment:
I don't think we should talk about `DELETE` explicitly.

Reply:
Removed it.